pkg/tuple/tuple.go

```diff
@@ -55,6 +55,14 @@ func BuildObject(objectType, objectID string) string {
 	return fmt.Sprintf("%s:%s", objectType, objectID)
 }
 
+// GetObjectRelationAsString returns a string like "object#relation". If there is no relation it returns "object"
+func GetObjectRelationAsString(objectRelation *openfgapb.ObjectRelation) string {
```

Comment:
You are introducing a new function, but not using it anywhere?

Reply:
It's used in Auth0 FGA :)

Reply:
Thank you! Yes, we talked offline about it. My personal opinion is that in that case it shouldn't be here, but I will defer to others.

Reply:
There are other places in our code within OpenFGA that should delegate to this function as well. Here is a list of them that I found from a quick search (there may be others):

* `pkg/server/commands/connected_objects.go` (line 400)
* `pkg/storage/memory/memory.go` (line 387)

Please refactor those to use this method as well. Then this debate is non-applicable 😄

Reply:
Done!  Thanks - i don't know why I didn't check where I could use this here, that was dumb..